mobile/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "mobile",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
```

Comment:
Nitpick, but can we please have the same ordering in `package.json` fields as the other examples?